tests/NumberTest.php

```diff
@@ -374,8 +444,12 @@ public function provideRoundingTestCases()
             'truncate 6'  => ['1.23456789', 6, Rounding::ROUND_TRUNCATE, '1.234567'],
             'truncate 7'  => ['1.23456789', 7, Rounding::ROUND_TRUNCATE, '1.2345678'],
             'truncate 8'  => ['1.23456789', 8, Rounding::ROUND_TRUNCATE, '1.23456789'],
+            // adds trailing zeroes
             'truncate 9'  => ['1.23456789', 9, Rounding::ROUND_TRUNCATE, '1.234567890'],
             'truncate 10' => ['1.23456789', 10, Rounding::ROUND_TRUNCATE, '1.2345678900'],
+            // keeps trailing zeroes
+            'truncate zeroes 1' => ['1.00000001', 3, Rounding::ROUND_TRUNCATE, '1.000'],
```

Comment:
@Quetzacoalt91 This test verifies the fix of the problem that you pointed out on this PR: https://github.com/PrestaShop/PrestaShop/pull/8238